numba/tests/test_array_analysis.py

```diff
@@ -354,7 +354,7 @@ def test_12():
             c = a[1:,:]
             d = b[:-1,:]
             e = c.shape[0]
-            f = d.shape[0]
+            f = len(d)
```

Comment:
Perhaps this could be `g = len(d)` and then use it in the return, and then assert equivalence over `'e', 'f', 'g'` so as to still retain the original check based on shapes and slices in the 2D array context?

Reply:
done.